### PR TITLE
Change ticket processing page to display comment content below user info on small phones

### PR DESCRIPTION
**Pull request by mimi89999**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | issue wasn't reported

Before:
Capture d’écran - 2020-04-28 à 10 00 25
Capture d’écran - 2020-04-28 à 10 30 29

Alter:
Capture d’écran - 2020-04-28 à 09 59 26
Capture d’écran - 2020-04-28 à 10 34 59

### CODE REVIEW

**Comment by mimi89999** [comment]
@trasher Could you please review this?

File: css/styles.scss [hunk 1]
```diff
@@ -3745,6 +3745,8 @@ a.copyright {
       clear: both;
       padding-top: 8px;
       border-top: 1px dashed #D2D2D2;
+      display: flex;
+      flex-flow: row wrap;
```
Comment (orthagh) [inline css/styles.scss:3749]: this property broke right displayed items (not related to this pr, but i think adding right items is currently broken in 9.5/bugfixes, will check this afternoon).

You should change add to h_item a specific rule like this i think:
```scss
   .h_item {
      ...

      &.right {
         flex-direction: row-reverse;
      }
   }
```

Also, moving to flex layout seems to have sub divs to have 100% height of their container, don't know why

image

Comment (orthagh) [reply css/styles.scss:3749]: at least for the 100% height, i think we cant to push status bar to the bottom

Comment (orthagh) [reply css/styles.scss:3749]: nevermind my comment on adding right item, bug was on my side only

Comment (mimi89999) [reply css/styles.scss:3749]: > nevermind my comment on adding right item, bug was on my side only

What should I change then?

Comment (orthagh) [reply css/styles.scss:3749]: you should adding the `&.right` rule

i was taking about 
> (not related to this pr, but i think adding right items is currently broken in 9.5/bugfixes, will check this afternoon)

Comment (mimi89999) [reply css/styles.scss:3765]: Ah. I see the issue now. Added that CSS. Thanks!

**Comment by trasher** [comment]
Thank you :)

**Pull request merged**